custom_components/babybuddy/services.yaml

```diff
@@ -147,143 +93,184 @@ services_feedings_add:
       name: Feeding amount
       description: Feeding amount in number format
       selector:
-        text:
+        number:
+          min: 1
+          max: 300
+          mode: box
     notes:
       name: Feeding notes
       description: Feeding notes
       selector:
         text:
+          multiline: true
 
-services_notes_add:
-  name: Add Note
-  description: Adds a note
+add_sleep:
+  name: Add Sleep
+  description: Adds sleep entry
+  target:
+    entity:
+      integration: babybuddy
+      domain: switch
```

Comment:
should this be switch?

Reply:
`add_sleep` service is part of the switch entity because a feeding can be linked to a timer (which is a switch in HA). If timer option is enabled in the service call the timer ID is used in the timer field in the api call. This allows to automatically assign the start/end times based on the active timer.